### DISCUSSION TITLE
How to get an older version of the loader

### DISCUSSION THREAD

**Discussion by partinparticular** [Epix-Incorporated/Adonis#1795] [General]
Not so long ago, the modern theme changed from what was a very **very** good looking modern theme to a much less modern-looking theme.. I'm wondering if I can do anything to get the older version of the loader to get this better modern theme.

**Comment by Expertcoderz** [comment]
No, you can't. The built-in themes are located in the MainModule, not the loader. If you don't like the theme in its current form, the only way is to create a custom theme and put it in the loader, or choose a different one from the built-in themes.
